### PR TITLE
Add  configuration option to enable/disable o-errors

### LINKED ISSUE #10: Add stub methods for when raven not initialised
As discussed on slack:

Rhys Evans [9:46 AM] 
@samuel.giles: I'm looking into using o-errors in next. One feature our current implementation has is that raven can be turned off using a feature flag, but the methods are all mocked in that case and can still be called without erroring. I could wrap o-errors to preserve this functionality, but is there value in having similar functionality built into o-errors

Sam Giles [11:33 AM] 
wheresrhys: I think adding it into o-errors makes sense.  At the moment you can't really turn off error tracking, you either have tracking or you don't (the best you can do at the moment is 'not' call `init`).  You opt in to logging.

### PR DESCRIPTION
**Pull request by samgiles**
Closes #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,16 @@ respectively.  This should not be used in production.
 }
 ```
 
+##### disabled     - optional
+
+If `true` `oErrors` will be configure with no operation functions and errors
+will not be reported.
```
Comment (triblondon) [inline README.md]: A few typos.  Also consider making it `enabled` with a default value of true rather than the other way around.

Comment (samgiles) [inline README.md]: I consciously chose disabled for a couple of reasons, the main reason is by default I want it to be enabled, and disabled only by explicitly setting `disabled = true`. I'm happy to change as long as the default is still on when `options.enabled` is undefined.

**Comment by wheresrhys** [comment]
What's this difference between this and `logLevel: off`. Also, can it please be released today if practical - I'm almost done refactoring our flags client and would like to add something similar to this on the client side too https://github.com/Financial-Times/next-feature-flags-client/blob/master/server.js#L14

**Comment by samgiles** [comment]
Logging is a separate API: https://github.com/Financial-Times/o-errors#log-data-and-associating-logs-with-errors

Will do.

**Pull request merged**